src/commands/env.rs

```diff
@@ -44,6 +46,17 @@ fn make_symlink(config: &FnmConfig) -> std::path::PathBuf {
     temp_dir
 }
 
+fn merge_path(path: &Path) -> Result<String, Error> {
+    let current_path = std::env::var_os("PATH").ok_or(Error::CantReadPath)?;
+    let mut split_paths: HashSet<_> = std::env::split_paths(&current_path)
+        .filter(|p| !p.to_str().unwrap().contains("fnm_multishells"))
```

Comment:
👀 what if the name would not be `fnm_multishells`? or someone will have a `/home/user/fnm_multishells_are_annoying/my-bin`? 😜

Maybe we should filter out the input `FNM_MULTISHELL_PATH` and then we can call it a day? https://github.com/Schniz/fnm/blob/74f60b3590ce93d63e28349cb4ca0116cf23702c/src/config.rs#L33

I'm thinking here with you, I don't have the answers hehe 😃 
again, sorry if that's annoying.

Reply:
Does it possible? Because every time `env` was invoked the temp dir will be created based on `$TMPDIR/fnm_multishells`
https://github.com/Schniz/fnm/blob/52a9cfedd1327b9d7a828d8ecc1b84077ca8c6fa/src/commands/env.rs#L35-L37
IMO Filter it out by using the value from `std::env::temp_dir().join("fnm_multishells")` is better

Reply:
Well we can do that for now. When I land the location change I will have to change this line to conform the new location too but that's cool.